CourseApp/Calculation.cs

```diff
@@ -0,0 +1,45 @@
+namespace CourseApp
+{
+    using System;
+    using System.Collections.Generic;
+    using static System.Math;
+
+    public class Calculation
+    {
+        public void TaskA(double a, double xn, double xk, double dx)
```

Comment:
Can we move specific tasks from `Calculation` class? It should not known about them

Reply:
Done,